cfme/services/myservice/ssui.py

```diff
@@ -329,12 +325,13 @@ def step(self):
 
 @navigator.register(MyService, 'Retire')
 class MyServiceRetire(SSUINavigateStep):
-    VIEW = RetireServiceView
+    # TODO - remove check when BZ 1653288 is fixed
```

Comment:
This should be done dynamically based on `BZ(1653288).blocks`, otherwise tests will pass now and fail when the BZ is fixed because they'll have the wrong view again.

Ultimately I don't think this should really be changed in the framework to account for the BZ.  The framework should reflect, as much as possible, the application's intended design.  I believe we should avoid coding workarounds into our modeling, especially statically for BZ's.  

I think these changes are confusing a few different things:

1. Is the page displayed on navigation via selecting `Lifecycle->Retire` really version dependent, or is it BZ dependent?
2a. If the view is version dependent, then this should stay
2b. If the view is BZ dependent, this should not be changed, it should be the view that _should_ be displayed
3. Handling of the wrong view displayed due to BZ should be in `retire` method, not navigation
4. Tests could be skipped instead based on BZ, instead of writing workaround ( 3 ) into the retire method